app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,11 @@
+class GuestRoomsController < ApplicationController
+  def create
+    guest = Guest.find(params[:id])
+    room = Room.find(params[:add_room_id])
+
+    guest.rooms.push(room)
+    guest.save
```

Comment:
I believe (but it might be worth checking) that if you add a room to the guest's room collection, it should save automatically. But I could be wrong! Under the hood, when you add to the rooms collection, Rails is creating a joins record (GuestRoom.create(room_id: room_id, guest_id: guest_id)) rather than just instantiating a joins record. Will double check this